runtime.md

```diff
@@ -66,13 +65,15 @@ This operation MUST generate an error if it is not provided the ID of a containe
 This operation MUST return the state of a container as specified in the [State](#state) section.
 In particular, the state MUST be serialized as JSON.
 
+After a user launches a container using the [start](#start) operation, state requests by the same user in the same namespaces with the same runtime installation MUST successfully return that container's state by the time the [pre-start hooks](config.md#prestart) are called (step (3) in the [lifecycle](#lifecycle)) through the final [post-stop hooks](#config#poststop) (step (8) in the [lifecycle](#lifecycle)).
+Runtimes MAY return state under other conditions, for example to a different user or when asked from a different namespace.
```

Comment:
This text should just be focused on scope of access to the ops - its not really about state. GeState() is just one op, this access stuff applies to all ops.